tcs-persistence/src/test/java/com/transformuk/hee/tis/tcs/service/repository/ProgrammeMembershipRepositoryIntTest.java

```diff
@@ -0,0 +1,76 @@
+package com.transformuk.hee.tis.tcs.service.repository;
```

Comment:
It is a new file. Just wondering do we need to have the license declaration at the beginning?

Reply:
I can't see any license declaration in any of other tcs files, so think it's not needed.